js/src/common/components/ModalManager.js

```diff
@@ -16,6 +16,12 @@ export default class ModalManager extends Component {
     );
   }
 
+  onupdate() {
+    if (this.$('Modal') && !this.attrs.state.modal) {
```

Comment:
Are you sure about this? Shouldn't it be `.Modal` ?

Reply:
Should be... Probably I made a typo when extracting it from the main PR, thanks for the catch!

Reply:
So... was it an error or not? I see you did not change it

Reply:
Now I'm confused... Code editor says I pushed that commit...

Reply:
Turns out I pushed to the wrong remote... Good job me! :see_no_evil: